x-pack/plugin/ql/src/main/java/org/elasticsearch/xpack/ql/type/DateUtils.java

```diff
@@ -52,6 +52,14 @@ public static ZonedDateTime asDateTime(long millis) {
         return ZonedDateTime.ofInstant(Instant.ofEpochMilli(millis), UTC);
     }
 
+    public static ZonedDateTime asDateTime(long millis, ZoneId zoneId) {
```

Comment:
Since you always pass UTC, why not use the above?

Reply:
Remanent from a first attempt to use session's zoneID. Updated to use the existing one.